packages/samples/headless-react/src/engine.ts

```diff
@@ -1,6 +1,13 @@
-import {HeadlessEngine, searchAppReducers} from '@coveo/headless';
+import {
+  HeadlessEngine,
+  searchAppReducers,
+  recommendationAppReducers,
+} from '@coveo/headless';
 
 export const engine = new HeadlessEngine({
   configuration: HeadlessEngine.getSampleConfiguration(),
-  reducers: searchAppReducers,
+  reducers: {
+    ...searchAppReducers,
+    ...recommendationAppReducers,
```

Comment:
So the intent when we created the Recommendation controller was to use it with a dedicated engine. In retrospect, this may not have been a good design choice, but for now I would suggest creating a dedicated engine.

I think unintended interactions could occur if using a recommendation controller with a search app controller and giving both the same engine. I created a spike in our backlog: https://coveord.atlassian.net/browse/KIT-455